printer_mods/Dr-Info/corner_cable_cover_with_drop_down_holes/README.md

```diff
@@ -1,27 +1,44 @@
 # Top Corner Cable Cover with Drop Down Holes (LED Wires)
-Covers cables that are running around the top corners of 2.4R2 extrusions behind the Z idlers with drop down holes around the vertical extrusions and inline holes with cutouts to keep the wire management as clean as possible.
 
-Holes are big enough to allow 2 pin microfit connectors pass through. Works perfectly with LDO LED PCB breakout board.
+This is originally based off of samwiseg0 mod 
+<br /> https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/samwiseg0/corner_cable_hide
+
+Modified LDO's PCB splitter spacer
+<br /> https://github.com/MotorDynamicsLab/LDOVoron2/blob/main/STLs/led_pcb_spacer.stl
```

Comment:
Please avoid using html in markdown files.  The <br /> tags can be replaced with a blank line